### ISSUE TITLE
[12.5 -> 16.2] Custom modifiers now ignored

### ISSUE THREAD

**Issue by pokonski** [airbnb/react-dates#1017]
Hey guys!

I am in the process of migrating the old 12.5.1 version into 16.2.0 and I noticed that modifiers are now hardcoded here inside CalendarDay component: https://github.com/airbnb/react-dates/blob/9c01aba9a2601f6a38c2910ec94c6194a07ff467/src/components/CalendarDay.jsx#L131-L142

Is it a regression or an intentional change? I had a custom object with my own additional modifiers which are now ignored.

**Comment by ljharb** [comment]
v12 to v16 is 4 sets of breaking changes; it’s not a regression.

**Comment by pokonski** [comment]
@ljharb yeah I realize that, I was asking because I wanted to know if you are open to a PR which will restore custom modifiers.

**Comment by majapw** [comment]
hi @pokonski! Can you give us some context on what kind of custom modifiers you were using? That might give us a bit more information to make a call. :)

**Comment by pokonski** [comment]
Hey @majapw, absolutely. One of the things we use them for is coloring various days depending on how _busy_ they are.

We use the always visible date picker next to a calendar so users can know which days have anything in them, for example `green` background means that there are time slots available to schedule an event. `Red` means it's a fully booked day, etc.

Right now I managed to "recycle" some of the other hardcoded modifier names for our purpose but would be good to have a clean one to define them again.
